Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
+ARG PYTHON_VERSION=3.9
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt install -yq  software-properties-common
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install -yq python${PYTHON_VERSION} python3-pip python${PYTHON_VERSION}-dev python3-venv python${PYTHON_VERSION}-distutils python3-psutil
+RUN apt-get install -yq git build-essential liblz4-dev libunwind-dev
+
+RUN pip install pipx
+RUN which pipx
+ENV PATH=/root/.local/bin:$PATH
+RUN pipx install ddev
+# Install ruff to apply linter recommandation: ruff --fix
+RUN pipx install ruff
```

Comment:
I would recommend not using pipx but rather a virtual environment. also you would want the local version of ddev by passing the path to `pip install -e ...`

There is no need for Ruff because `ddev test` already manages that for you

Reply:
indeed `ddev test` manage ruff for us. 
but usually us `ruff` directly to run the "autofix" command. 

I haven't found an option in `ddev` to fix automatically linter recommandation. does it exist?

Reply:
Ok I founded it `ddev test -fs`